src/Microsoft.DotNet.Watcher.Tools/CommandLineOptions.cs

```diff
@@ -26,7 +26,17 @@ public static CommandLineOptions Parse(string[] args, TextWriter stdout, TextWri
                 FullName = "Microsoft DotNet File Watcher",
                 Out = stdout,
                 Error = stderr,
-                AllowArgumentSeparator = true
+                AllowArgumentSeparator = true,
+                ExtendedHelpText = @"
+Remarks:
+  The argument separator '--' can be used to ensure separation of the
+  arguments for 'dotnet-watch' and those passed into the subprocess.
+  For example: dotnet watch --quiet -- test -class TestClass1
+
+Examples:
+  dotnet watch run
+  dotnet watch test
```

Comment:
I think this is a little confusing in that in the first example it says `-- test`, but here it's just `test`. How could you make it clearer that both work the same way?